priv/repo/seeds.exs

```diff
@@ -1,11 +1,19 @@
-# Script for populating the database. You can run it as:
-#
-#     mix run priv/repo/seeds.exs
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     Katana.Repo.insert!(%Katana.SomeSchema{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+defmodule Atomic.Repo.Seeds do
+  @moduledoc """
+  Script for populating the database.
+  You can run it as:
+    $ mix run priv/repo/seeds.exs # or mix ecto.seed
+  """
+  @seeds_dir "priv/repo/seeds"
+
+  def run do
+    [
+      "user.exs"
```

Comment:
This file should be called `accounts.exs` in order to follow Elixir naming conventions.